spec/mailers/user_mailer_spec.rb

```diff
@@ -533,138 +533,196 @@ def expect_email_body_to_have_help_and_contact_links
     end
   end
 
-  describe '#in_person_ready_to_verify' do
+  context 'in person emails' do
```

Comment:
@JackRyan1989 -
One final thought- maybe overkill. Sorry I came back to this. Looking inside user_mailer_spec.rb...  I feel like there are 4 scenarios here- and we are testing only 2.  I am not understanding tests correctly. Thoughts?

Current Address Match Enabled / Same Address (Double Address Verf / Text Present
1. t/t/f
2. t/f/f
3. f/t/f (test line 568)
4. f/f/t (test line 559)

Reply:
@gina-yamada You're correct that there are only two cases here, and that's because I'm only testing if DAV is enabled or not. I'm not doing the testing for same address as id because the presenter specs should handle those cases. 

I could add in test cases for same_address_as_id but I don't think that should hold up this PR. Let me know if you disagree!

Reply:
@JackRyan1989  I agree!